### PR TITLE
Another docs cleaning PR

### PR DESCRIPTION
**Pull request by bolinocroustibat**
Minor docs fix PR:

- Fix wrong GitHub links in docs
- Fix wrong Python version in docs

Can be left as draft for later cleaning tasks if needed.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ We’ll use the following repositories:
 # Check the system requirements
 
 !!! info
-    Be aware that udata now requires Python **>3.7,<3.10** to work. 
+    Be aware that udata now requires Python **>3.7,<=3.11** to work.
```
Comment (ThibaudDauce) [inline docs/getting-started.md:13]: I don't think we support 3.7 anymore :-) I think it's at least 3.9

File: udata/settings.py [hunk 2]
```diff
@@ -132,7 +132,7 @@ class Defaults(object):
     SITE_KEYWORDS = ['opendata', 'udata']
     SITE_AUTHOR_URL = None
     SITE_AUTHOR = 'Udata'
-    SITE_GITHUB_URL = 'https://github.com/etalab/udata'
+    SITE_GITHUB_URL = "https://github.com/opendatateam/udata"
```
Comment (maudetes) [inline udata/settings.py:135]: Single quotes are preferred (for now :p)
```suggestion
    SITE_GITHUB_URL = 'https://github.com/opendatateam/udata'
```

File: docs/getting-started.md (hunk 1 above)
Comment (bolinocroustibat) [reply docs/getting-started.md:13]: Great, updated.

File: udata/settings.py (hunk 2 above)
Comment (bolinocroustibat) [reply udata/settings.py]: Fixed

**Pull request merged**